README.md

```diff
@@ -21,6 +21,18 @@ LIGHTUP_TLA=dum LIGHTUP_TOKEN=e3976956-9ef0-44c5-a978-2ee9149e1234 LIGHTUP_INSTA
 
```

Comment:
The bootstrap command will no go through as it stands if the user needs proxy tunneling, so this is confusing. 

I'd suggest wrapping up the proxy choice into the bootstrap command, so the user can run it as:

#no proxy:
LIGHTUP_TLA=dum LIGHTUP_TOKEN=e3976956-9ef0-44c5-a978-2ee9149e1234 LIGHTUP_INSTALL=0 ./bootstrap.sh

#use proxy:
LIGHTUP_TLA=dum LIGHTUP_TOKEN=e3976956-9ef0-44c5-a978-2ee9149e1234 LIGHTUP_INSTALL=0 USE_PROXY=1 PROXY_HOST=proxy.dum.com PROXY_PORT=8192 DESTINATION_PORT=443 ./bootstrap.sh

Reply:
I think this feels more complicated - it might be easier to break this down into 2 steps. 
Can I think about how to simplify this in another PR? I think what we have now will work for fujitsu.